banyand/liaison/grpc/registry_test.go

```diff
@@ -193,12 +200,16 @@ func setupForRegistry() func() {
 		preloadStreamSvc,
 		tcp,
 	)
+	moduleDeferFunc := func() {
```

Comment:
Could you remove this defer function? If the repo uses the lifecycle framework to stop, `deferFunc` will help stop this repo. You don't have to add a separate one.